### ISSUE TITLE
Capitalization Threshold for Fixed Assets

### ISSUE THREAD

**Issue by matthewdwatkins** [frappe/erpnext#27533]
Hello community and maintainers. Thank you so much for such a wonderful piece of software.

In my country, we are allowed to choose a price per item threshold below which tangible asset purchases are posted against expense accounts. These purchases are never entered into asset accounts and are not depreciated. If we do this for one asset below the threshold we are required to do it for all.

Right now, ERPNext will not let you use the purchasing workflow for fixed assets without posting them to an asset account. I investigated using non-stock items to handle this use case, but determined that it wouldn't work. I need to be able to track custody, repairs, and maintenance from within the system, so they have to be fixed assets.

I've created a working patch for this feature to use with my organization but am concerned about maintainability. So, I'm wondering if there is any interest in a PR for something like this among the general community.

Basically, what the patch does is:

1. Add an Accounts Setting option to set the capitalization threshold if desired.
2. Change Purchase Invoice behavior to allow item expense heads to be set to Expense type accounts for fixed assets below the threshold only.
3. Prevent the buying controller from creating balance sheet GL Entries (CWIP & fixed asset account) for expensed assets. Only income statement entries (receivables and expenditure) are made.
4. Prevent the system from creating balance sheet GL entries when the expensed asset is sold or scrapped. It does make income statement entries (payables and income) for the sale price of the asset.

I would be very happy to share my code and/or discuss this idea further.
